### DISCUSSION TITLE
Admin Claims?

### DISCUSSION THREAD

**Discussion by Noah104** [TownyAdvanced/Towny#4646] [Q&A] [answered]
Is there a way to claim land as a admin like not for myself but for the server, like I want to claim a few chunks that are a spawn/warp wild.

**Comment by LlmDl** [comment, marked as answer]
The [How Towny Works' Mayor section](https://github.com/TownyAdvanced/Towny/wiki/How-Towny-Works#mayors) has an explanation on how to control 2 towns.
